sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -280,21 +280,21 @@ When deleting a key asynchronously before you purge it, you can await the `WaitF
 By default, this loops indefinitely but you can cancel it by passing a `CancellationToken`.
 
 ```C# Snippet:DeleteAndPurgeKeyAsync
-DeleteKeyOperation operation = client.StartDeleteKey("key-name");
+DeleteKeyOperation operation = await client.StartDeleteKeyAsync("key-name");
```

Comment:
You can't just rewrite the code. These come from sample source snippets, which I don't see in this code. This will just be overwritten - everything in these files from source - unless you update the corresponding sample source. Find the name of the region "Snippet:DeleteAndPurgeKeyAsync" to do so.